stylecop.ruleset

```diff
@@ -42,10 +44,12 @@
     <Rule Id="CA1405" Action="Warning" />
     <Rule Id="CA1410" Action="Warning" />
     <Rule Id="CA1415" Action="Warning" />
+    <Rule Id="CA1806" Action="None" />
```

Comment:
xml comment for what CA1806 is? when Action="None"

Reply:
add comment